AppSettingsManagerBff/AppSettingsManagerBff.Infrastructure/ApiRepositories/HttpSettingsRepository.cs

```diff
@@ -1,19 +1,13 @@
-using System.ComponentModel.DataAnnotations;
-using System.Text.Json;
 using AppSettingsManagerBff.Domain.ApiRepositories;
-using AppSettingsManagerBff.Model.Api;
+using AppSettingsManagerBff.Model;
+using AppSettingsManagerBff.Model.Requests;
 
 namespace AppSettingsManagerBff.Infrastructure.ApiRepositories;
 
 public class HttpSettingsRepository : ISettingsRepository
 {
     private readonly HttpClient _httpClient;
```

Comment:
I might update this with the factory pattern I did for assignment 4. But totally fine for now. Pretty much inconsequential